### PR TITLE
Changelog 0.15b

### PR DESCRIPTION
**Pull request by phadej**
- https://hackage.haskell.org/package/servant-0.15/candidate
- https://hackage.haskell.org/package/servant-server-0.15/candidate
- https://hackage.haskell.org/package/servant-client-0.15/candidate
- https://hackage.haskell.org/package/servant-client-core-0.15/candidate
- https://hackage.haskell.org/package/servant-docs-0.11.3/candidate
- https://hackage.haskell.org/package/servant-foreign-0.15/candidate

new packages:

- https://hackage.haskell.org/package/servant-machines-0.15/candidate
- https://hackage.haskell.org/package/servant-pipes-0.15/candidate 
- https://hackage.haskell.org/package/servant-conduit-0.15/candidate

### CODE REVIEW

**Comment by phadej** [comment]
I'll look through this at the morning again.

**Comment by phadej** [comment]
... and if it still looks alright, then I'll cut the release branch

File: servant-client-core/servant-client-core.cabal [hunk 1]
```diff
@@ -1,27 +1,30 @@
+cabal-version:       >=1.10
 name:                servant-client-core
 version:             0.15
+
 synopsis:            Core functionality and class for client function generation for servant APIs
+category:            Servant, Web
```
Comment (alpmestan) [inline servant-client-core/servant-client-core.cabal]: Oh we get our own category now? =)

File: servant-client/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,72 @@
...
+0.15
+----
+
+- Streaming refactoring.
+  [#991](https://github.com/haskell-servant/servant/pull/991)
+  [#1076](https://github.com/haskell-servant/servant/pull/1076)
+  [#1077](https://github.com/haskell-servant/servant/pull/1077)
+
+  The streaming functionality (`Servant.API.Stream`) is refactored to use
+  `servant` own `SourceIO` type (see `Servant.Types.SourceT` documentation),
```
Comment (alpmestan) [inline servant-client/CHANGELOG.md]: `servant`**'s**

File: servant-client-core/servant-client-core.cabal (hunk 1 above)
Comment (phadej) [reply servant-client-core/servant-client-core.cabal]: we had it already, since 0.13 or 0.14: http://hackage.haskell.org/packages/#cat:Servant

**Pull request merged**